pyscriptjs/src/main.ts

```diff
@@ -3,6 +3,7 @@ import './styles/pyscript_base.css';
 import { loadConfigFromElement } from './pyconfig';
 import type { AppConfig } from './pyconfig';
 import type { Runtime } from './runtime';
+import { version } from './runtime';
```

Comment:
I think that this const should not live in `runtime.ts` FWIW, it doesn't make sense.
Probably the best place is here, directly in `main.ts`

Reply:
I would agree, though it creates a couple of circular dependencies since so much imports from `main.ts`, including pyconfig and runtime. I wasn't sure how much of a sin that was in TS, so I left it, but I won't be at all offended if it moves over.